stl/inc/__msvc_system_error_abi.hpp

```diff
@@ -0,0 +1,43 @@
+// __msvc_system_error_abi.h internal header (core)
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef __MSVC_SYSTEM_ERROR_ABI_HPP
+#define __MSVC_SYSTEM_ERROR_ABI_HPP

```

Comment:
Hmmm should we revisit the convention? I like the simplicity of the macro being literally the filename with . replaced with _ and then made all caps (in particular, as opposed to our existing standard vs. "x" header convention which I still view as an opaque mess).

Reply:
At a minimum, the `__msvc_meow` headers need to be consistent. Since there's only one current one, I think that changing it to double underscores would be reasonable.

Changing the guards for the `<vector>` family needs an issue and a separate overhaul. It will also be an obnoxious breaking change (IIRC there are some evil codebases that either sense whether STL headers have been dragged in, or define their guard macros).

*Or* we could file an issue to just `#pragma once` the whole world.

Reply:
Right, I mean only the other double underscore one, because when I added it I don't think I paid much attention to a convention to establish going forward.